packages/fast-components-react-base/src/dialog/dialog.spec.tsx

```diff
@@ -263,4 +263,37 @@ describe("dialog", (): void => {
 
         document.body.removeChild(container);
     });
+
+    test("should invoke focusTargetOnClose function on unmount of non-modal dialogs", () => {
```

Comment:
Should we also test focus invocation of element / ref<htmlelement> props?

Reply:
added one for elements